xml/html_title.xml

```diff
@@ -2735,4 +2736,23 @@
     <param pos="0" name="os.product" value="Pulse Connect Secure"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^Jellyfin$">
+    <description>Jellyfin media server</description>
+    <example>Jellyfin</example>
+    <param pos="0" name="service.vendor" value="Jellyfin"/>
+    <param pos="0" name="service.product" value="Media Server"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:jellyfin:jellyfin:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^proxmox - Proxmox Virtual Environment$">
+    <description>Proxmox open-source virtualization platform</description>
+    <example>proxmox - Proxmox Virtual Environment</example>
+    <param pos="0" name="service.vendor" value="proxmox"/>
+    <param pos="0" name="service.product" value="proxmox"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:proxmox:proxmox:-"/>
```

Comment:
I'm working on the fixup for this PR.  The CPE here and with Jellyfish will be removed by the CPE script because they are not valid, NIST issued CPEs.

https://github.com/rapid7/recog/blob/master/CONTRIBUTING.md#updating-cpes